tests/commands/test_alias.py

```diff
@@ -26,6 +26,12 @@ def projects_db(data_dir):
     project.activities.append(Activity(1, 'activity 1'))
     projects_list.append(project)
 
+    project = Project(45, '3rd active project with one inactive activity', Project.STATUS_ACTIVE)
```

Comment:
Could you please add a specific test that checks that inactive activities show up in red? You can do this by adding a `color` attribute to the `cli` function and pass it to `CliRunner.invoke` like so:

```python
diff --git a/tests/commands/conftest.py b/tests/commands/conftest.py
index 56177b2..d5ef961 100644
--- a/tests/commands/conftest.py
+++ b/tests/commands/conftest.py
@@ -135,7 +135,7 @@ def entries_file(tmpdir, config):

 @pytest.fixture
 def cli(config, data_dir):
-    def inner(cmd, args=None, input=None, return_stdout=True):
+    def inner(cmd, args=None, input=None, return_stdout=True, color=False):
         if not args:
             args = []

@@ -145,7 +145,7 @@ def inner(cmd, args=None, input=None, return_stdout=True):

         runner = CliRunner()
         result = runner.invoke(
-            taxi_cli, args, input=input, standalone_mode=False, catch_exceptions=False
+            taxi_cli, args, input=input, standalone_mode=False, catch_exceptions=False, color=color
         )

         return result.output if return_stdout else result
```

Reply:
Good suggestion, thanks!